src/core/server/http/router/route.ts

```diff
@@ -144,6 +152,24 @@ export interface RouteConfigOptions<Method extends RouteMethod> {
    * Additional body options {@link RouteConfigOptionsBody}.
    */
   body?: Method extends 'get' | 'options' ? undefined : RouteConfigOptionsBody;
+
+  /**
+   * Timeouts for processing durations.
+   * Default value: { server: false }
+   */
+  timeout?: {
+    /**
+     * Response timeout in milliseconds. Sets the maximum time allowed for the server to respond to an incoming request before giving up and responding with a Service Unavailable (503) error response.
+     * Default value: false.
+     */
+    server?: boolean | number;
```

Comment:
What does `false` exactly does? what about `true`? `hapi` doc is not really explicit on this.

Reply:
False will disable the timeout completely. The timeout right now is false when someone does not set it meaning that the server can take an infinite amount of time to process a request once it has received it.

Reply:
Setting this to true is a mistake in the Typings and docs from hapi, I will change this type to be like the other where it should be `false | number`